lib/server.go

```diff
@@ -2417,6 +2428,9 @@ func (srv *Server) _handleBlockBundle(pp *Peer, bundle *MsgDeSoBlockBundle) {
 		// gracefully fail.
 		srv._handleBlock(pp, blk, ii == len(bundle.Blocks)-1 /*isLastBlock*/)
 		numLogBlocks := 1000
+		if srv.params.IsPoWBlockHeight(blk.Header.Height) {
+			numLogBlocks = 25
```

Comment:
Do you want to leave this numLogBlocks at 25? Seems fine, just flagging in case you did it for debugging only.

Reply:
What would be a better value for PoW? we're doing bundles of 250. Maybe 50 instead? I want to see something in the logs at least 😂